### DISCUSSION TITLE
How can I find "detailedForecast" info in hourly weather API?

### DISCUSSION THREAD

**Discussion by STDataAndInformationService** [weather-gov/api#305] [Q&A]
**NOTE**
Please do not report operational issues (such as outages) here. We can not support these issues via this forum. Please report these as described here: https://weather-gov.github.io/api/reporting-issues

**Please describe what you're trying to do.**
A clear and concise description of what your issue is. Ex. I'm looking for forecast data...

In the hourly API, "detailedForecast" is blank. Is there way to get info?  I can get it from /gridpoints/{wfo}/{x},{y}/forecast. But it is not what I need.

Just double check, for the weather forecast, there are only two API

/gridpoints/{wfo}/{x},{y}/forecast
/gridpoints/{wfo}/{x},{y}/forecast/hourly

Did I missed any other API?

Thanks,

**Comment by scadergit** [comment]
Hello, please provide the URL of the request you attempted to make, a snippet of the response, and the correlation id found in the response header.

**Comment by STDataAndInformationService** [comment]
scadergit,

here is the URL: https://api.weather.gov/gridpoints/SEW/125,66/forecast/hourly

Here is part of response:

 "periods": [
            {
                "number": 1,
                "name": "",
                "startTime": "2020-08-10T14:00:00-07:00",
                "endTime": "2020-08-10T15:00:00-07:00",
                "isDaytime": true,
                "temperature": 76,
                "temperatureUnit": "F",
                "temperatureTrend": null,
                "windSpeed": "7 mph",
                "windDirection": "NNW",
                "icon": "https://api.weather.gov/icons/land/day/skc?size=small",
                "shortForecast": "Sunny",
                "detailedForecast": ""

 "detailedForecast" is empty.  May be this is by design?

Thanks

**Comment by scadergit** [comment]
It would not be by design. I was able to reproduce. We'll look into it, but it might take some time.

**Comment by STDataAndInformationService** [comment]
Great, thanks you check into it.

**Comment by scadergit** [comment]
Turns out I read too quickly yesterday, there is no description for _hourly_. The property will be removed from that endpoint on a future release.

**Comment by STDataAndInformationService** [comment]
Thanks your additional info and efforts
